README.MD

```diff
@@ -126,6 +126,17 @@ You can also create reusable templates
     $ http -f POST <api-service-address>/api/v1/projects/<my-project>/anonymize-image detectionType='OCR' analyzeTemplateId='<my-analyze-template-name>' anonymizeImageTemplateId='<my-anonymize-image-template-name>' imageType='image/png' file@~/test-ocr.png > test-output.png
     ```
 
+***Sample 6*** 
+1. Create an anonymize json template, this will define the json schema to be analyzed and anonymized.
+   ```sh
+    echo -n '{"jsonSchema":"{\"Name\": \"<PERSON>\", "Address": "<LOCATION>}"}' | http <api-service-address>/api/v1/templates/<my-project>/anonymize-json/<my-anonymize-json-template-name>
```

Comment:
just a thought. will it be better if the api will be just 'anonymize' / 'analyze' and you can pass 'regular' or json text?
and the method will detect if its a json or not?

Reply:
you're right, but we need to make other changes in the api - we will do it separately